.changeset/stale-terms-march.md

```diff
@@ -0,0 +1,7 @@
+---
+"chainlink": patch
+---
+
+Bump libocr => fd3cab206b2ca3b7ff207996b95673b2d6303ec4
```

Comment:
Shouldn't we mention the new feature?

Reply:
What is the new feature?

Reply:
It's not being used at the moment. But if you want to say it you could say "introduces the ability to report GasPriceSubunits in the median plugin - currently value is ignored"

Reply:
I didn't realize it wasn't connected yet. No need to mention then I guess

Reply:
It sounds like something with poll time changed though?